stoix/systems/ppo/anakin/ff_ppo_continuous.py

```diff
@@ -511,7 +511,15 @@ def run_experiment(_config: DictConfig) -> float:
         logger.log({"timestep": t}, t, eval_step, LogEvent.MISC)
         if ep_completed:  # only log episode metrics if an episode was completed in the rollout.
             logger.log(episode_metrics, t, eval_step, LogEvent.ACT)
```

Comment:
Isn't a little bit ambiguous to call that ACTOR steps_per_seconds as it is capturing the "learn" function ?

Reply:
I'm not sure i fully understand. In the anakin systems, the "Actors" generate the data within the "learn" function. The amount of data produced / elapsed time. The only real issue is that the elapsed time includes the learn step so the actual actor steps per second would be faster if you only timed that specific chunk of code.

Reply:
Unless by saying capturing the learn function, you mean this issue:

> The only real issue is that the elapsed time includes the learn step so the actual actor steps per second would be faster if you only timed that specific chunk of code.

Reply:
That was my point, the actor sps include the learning (which include backprop + inference e.g. within the _actor_loss_fn or the _critic_loss_fn) It makes me think that the actual "acting" time is mostly composed of this learning component than anything else

Reply:
Yeah you are right. This is actually something necessary to fix. Its going to be annoying since it'll need to be fixed in every single file. Can you make an issue and then we can chat about it there.

Reply:
I would be happy to help you fix it, we just need to investigate how to do it for one file

Reply:
awesome. Thanks so much.